core/store/orm/orm_test.go

```diff
@@ -1017,27 +1017,31 @@ func TestBulkDeleteRuns(t *testing.T) {
 
 	// matches updated before but none of the statuses
```

Comment:
The intent of this test would be clearer if the comments above each stanza had " -- should (not) be deleted by bulk delete."